src/OpenTelemetry.Exporter.Zipkin/OpenTelemetry.Exporter.Zipkin.csproj

```diff
@@ -26,6 +26,10 @@
     <Compile Include="$(RepoRoot)\src\OpenTelemetry\Internal\TagTransformer.cs" Link="Includes\TagTransformer.cs" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Include="$(RepoRoot)\src\OpenTelemetry\Internal\UnconditionalSuppressMessageAttribute.cs" LinkBase ="Common" Condition ="'$(TargetFrameworks)' != 'netstandard2.0' or '$(TargetFrameworks)' != 'net462'" />
```

Comment:
@Yun-Ting Does "LinkBase" become the folder shown in IDE? If yes, I think typically we use "Includes" as the virtual folder thingy.

nit: There are a bunch of "Compile" definitions in the `ItemGroup` above, might be nice to add this one in that group instead of having another 🤷

Reply:
> Does "LinkBase" become the folder shown in IDE?

Nope. https://learn.microsoft.com/en-us/visualstudio/msbuild/common-msbuild-item-metadata?view=vs-2022

I updated to use Link here and put the `UnconditionalSuppressMessageAttribute.cs` under the Includes folder so it's consistent with what we already have. Thanks!